docs/UserGuide.md

```diff
@@ -62,7 +62,23 @@ Expected display for first time users:
 	Hi! You seem to be new, are you ready?!
 	Please enter your new account password:
 ```
-Expected output after successfully creating password: <br>
+To help users manage their finances well, users are also prompted to enter budgets with the following 3 questions,
+users should enter valid amounts for each budget type <br>
```

Comment:
Maybe can explain abit what is determined as valid

Reply:
editted